<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>waiterWindow</class>
 <widget class="QMainWindow" name="waiterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="box1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>桌1</string>
    </property>
    <widget class="QPushButton" name="waterBtn1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>加水</string>
     </property>
    </widget>
    <widget class="QPushButton" name="remindBtn1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>催菜</string>
     </property>
    </widget>
    <widget class="QPushButton" name="payBtn1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>结账</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>271</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>桌2</string>
    </property>
    <widget class="QPushButton" name="waterBtn2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>加水</string>
     </property>
    </widget>
    <widget class="QPushButton" name="remindBtn2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>催菜</string>
     </property>
    </widget>
    <widget class="QPushButton" name="payBtn2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>结账</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>461</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
